### PR TITLE
Create Jenkins pipeline to release the operator

### PR DESCRIPTION
**Pull request by israel-hdez**
Adds a Jenkins pipeline to release the operator.

Related back-end PR: kiali/kiali#3567
Related kiali/kiali#2895

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+ *      defaultValue: quay.io/kiali/kiali-operator
+ *      description: The name of the Quay repository to push the operator release
+ */
+
+def bumpVersion(String versionType, String currentVersion) {
+  def split = currentVersion.split('\\.')
+    switch (versionType){
+      case "patch":
+        split[2]=1+Integer.parseInt(split[2])
+          break
```
Comment (jmazzitelli) [inline Jenkinsfile:28]: Why does the indentation here look different than the other `case` statements?

**Review by jmazzitelli — DISMISSED** [review]
This looks good, but the indentation looks weird.
I did not run any of this.

File: Jenkinsfile (hunk 1 above)
Comment (israel-hdez) [reply Jenkinsfile:28]: It's VIM mixing tabs and white spaces when indenting. I'll fix.

**Comment by israel-hdez** [comment]
@jmazzitelli Indentation is fixed.
As cfcosta seems to be out, I'm wondering if you could also review the back-end repo PR: https://github.com/kiali/kiali/pull/3567

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**